app/assets/javascripts/discourse/app/components/user-menu/menu-tab.hbs

```diff
@@ -0,0 +1,31 @@
+{{#if this.renderAsLink}}
+  <a
+    class={{this.classNames}}
+    id={{this.id}}
+    tabindex={{this.tabIndex}}
+    aria-selected={{concat "" this.isActive}} {{! convert bool to string}}
+    aria-controls={{this.ariaControls}}
+    data-tab-number={{@tab.position}}
+    href={{@tab.linkWhenActive}}
+    {{did-insert this.focusLink}}
+  >
+    {{d-icon @tab.icon}}
+    {{yield}}
+  </a>
+{{else}}
+  <button
+    type="button"
+    role="tab"
+    class={{this.classNames}}
+    id={{this.id}}
+    tabindex={{this.tabIndex}}
+    aria-selected={{concat "" this.isActive}} {{! convert bool to string}}
```

Comment:
Never had this case, but wouldnt this be slightly more clear and avoid the comment?

```
aria-selected={{if this.isActive "true" "false"}}
```

Reply:
yeah this is much better, thanks!